src/main/java/seedu/partyplanet/logic/commands/DeleteCommand.java

```diff
@@ -8,10 +8,11 @@
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list\n"
+            + ": Deletes all contacts in currently filtered list\n"
+            + "OR Deletes the person identified by the index number used in the displayed person list\n"
             + "OR Delete the person with the provided tag,\n"
             + "unless the person has other tags, then only the tag is removed\n"
-            + "Parameters: {INDEX (must be a positive integer) [INDEX]... | -t TAG [-t TAG]...}\n"
+            + "Parameters: [{INDEX (must be a positive integer) [INDEX]... | -t TAG [-t TAG]...}]\n"
             + "Example: " + COMMAND_WORD + " 1 2 3";
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
```

Comment:
Perhaps a more natural phrasing?

```suggestion
    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted the following people: %1$s";
```

Reply:
```suggestion
    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted the following persons: %1$s";
```